static/js/pages/SitePage.tsx

```diff
@@ -32,13 +34,13 @@ export default function SitePage(): JSX.Element | null {
   }
 
   return (
-    <div className="site-page std-page-body container">
-      <h3>{website.title}</h3>
+    <div className="site-page std-page-body container pt-3">
       <div className="content-container">
         <Card>
           <SiteSidebar website={website} />
         </Card>
         <div className="content pl-3">
+          <h3 className="py-5">{website.title}</h3>
```

Comment:
Can we assert the title in `SitePage.test.tsx`?

Reply:
sure, I'll add that